test/integration/filter-email-addresses.test.js

```diff
@@ -252,4 +253,39 @@ describe('filter email addresses by event', () => {
 
     expect(result).toStrictEqual([...emailAddresses.flat(), ...splitDevEmailAddresses, ...splitRpaEmailAddresses])
   })
+
+  test('should return a flat array of devEmailAddresses when an event with name of "batch-processing-payment-request-invalid" is given', async () => {
+    const batchProcessingEvent = { ...DEV_EVENT }
+    const result = filterEmailAddresses(batchProcessingEvent)
+
+    expect(result).toStrictEqual(splitDevEmailAddresses)
+  })
+
+  test('should return a flat array of devEmailAddresses when an event with name of "responses-processing-quarantine-error" is given', async () => {
+    const batchProcessingEvent = { ...DEV_EVENT, name: 'responses-processing-quarantine-error' }
```

Comment:
@simonnineteen90 

Variable name here doesn't make sense as its not a batch processing error but a responses processing error.

Could use the reserved variable `event` here (see line 138 for inspo)

Reply:
updated